lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/HelloSimpleEventHandlers.cs

```diff
@@ -174,7 +174,7 @@ public void OnNext(ICompletedTask value)
         {
             using (Logger.LogFunction("HelloSimpleEventHandlers::CompletedTask received"))
             {
-                Logger.Log(Level.Info, string.Format(CultureInfo.InvariantCulture, "Received CompletedTask: {0}, task id: {1}", value.Id, _taskContext.CurrentTaskId()));
+                Logger.Log(Level.Info, string.Format(CultureInfo.InvariantCulture, "Received CompletedTask: {0}, task id: {1}, task value: {2}.", value.Id, _taskContext.CurrentTaskId(), ByteUtilities.ByteArrarysToString(value.Message)));
```

Comment:
Logging the message into the `Info` level seems too verbose. Maybe have a second or different log line for the `Verbose` level?